deku-derive/src/lib.rs

```diff
@@ -431,6 +432,9 @@ struct FieldData {
     /// tokens providing the number of bytes for the length of the container
     bytes_read: Option<TokenStream>,
 
+    /// read exactly that many bytes
+    read_exact: Option<TokenStream>,
```

Comment:
thinking about this now, could we not specialize `count` for `Vec<u8>` ?

Reply:
Yah, proc-macro specializations aren't my favorite, but we don't really have other options... Updated all the code